requirements.txt

```diff
@@ -14,3 +14,4 @@ httpx>=0.27.0,<1
 protobuf>=3.20.2 # not directly required, pinned by Snyk to avoid a vulnerability
 opal-common==0.7.6
 opal-client==0.7.6
+permit-datafilter>=0.0.3,<1
```

Comment:
make this "optional" dependency, otherwise you require everyone that uses our SDK to have SQLAlchemy and many other unnecessary packages